lib/tasks/mastodon.rake

```diff
@@ -792,15 +792,23 @@ namespace :mastodon do
 
         begin
           code = Request.new(:head, account.uri).perform(&:code)
-        rescue StandardError
+        rescue StandardError => e
           # This could happen due to network timeout, DNS timeout, wrong SSL cert, etc,
           # which should probably not lead to perceiving the account as deleted, so
           # just skip till next time
+          progress_bar.pause
```

Comment:
`progress_bar.pause`, `progress_bar.clear` and `progress_bar.resume` could be moved out of the conditionals I guess, as every codepath calls them

Reply:
The progress bar is slightly broken anyway - it tends to overflow its initial value towards the end, I assume because the instance learns about new accounts while the task is running.
I'll look at the pause/resume, but it may exceed my non-existing knowledge of the language.

Reply:
I just meant that those functions are currently called in the same order regardless of the branch that ends up being executed. Therefore, they could be moved outside of the conditional to make the code a bit cleaner.